src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,20 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      {post.user && <UserInfo user={post.user} />}
+    </div>
+    <p className="PostInfo__body">{post.body}</p>
+    <hr />
+    {post.comments ? (
```

Comment:
since comments is an array (we will not use `null`)
```suggestion
    {post.comments.length > 0 ? (
```